web-components/src/components/datepicker/datepicker-day/DatePickerDay.ts

```diff
@@ -59,16 +59,18 @@ export namespace DatePickerDay {
     }
 
     handleClick = (e: MouseEvent) => {
-      this.dispatchEvent(
-        new CustomEvent("day-select", {
-          bubbles: true,
-          composed: true,
-          detail: {
-            date: this.day,
-            sourceEvent: e
-          }
-        })
-      );
+      if (!this.disabled) {
```

Comment:
THis new check is required, was it a bug earlier?

Reply:
Removed this check. Not required here